file/src/main/scala/akka/stream/alpakka/file/scaladsl/Archive.scala

```diff
@@ -21,4 +21,10 @@ object Archive {
   def zip(): Flow[(ArchiveMetadata, Source[ByteString, Any]), ByteString, NotUsed] =
     ZipArchiveManager.zipFlow()
 
+  /**
+   * Flow for packaging multiple files into one TAR file.
+   */
+  def tar(): Flow[(ArchiveMetadataWithSize, Source[ByteString, Any]), ByteString, NotUsed] =
```

Comment:
```suggestion
  def tar(): Flow[(ArchiveMetadataWithSize, Source[ByteString, _]), ByteString, NotUsed] =
```

Reply:
I had this before, but I wanted to be consistent with the `Archive.zip` API. Is it okay to also change the API for `.zip()` so that both use `Source[ByteString, _]`?

Reply:
Changing `zip` wouldn't be fully backwards compatible. But you can do better on the new method.